schedule/yam/agama/add_home_file_system.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: add_home_file_system
+description: >
+  Playwright use add-home-file-system test on agama
```

Comment:
Please add a description independent of a name of a file that could potentially change.
s/agama/Agama/ (only for text)

Reply:
Updated